addons/project/static/src/js/tours/project.js

```diff
@@ -162,25 +162,34 @@ registry.category("web_tour.tours").add('project_tour', {
     position: "bottom",
     run: "click",
 },
-
 {
     trigger: ".o_form_project_tasks",
     isActive: ["auto"],
 },
 {
+    isActive: ["auto"],
     trigger: ".o_field_widget[name='user_ids'] input",
     content: _t("Assign a responsible to your task"),
     position: "right",
-    run: "edit Mitch",
+    run: "edit Admin",
 },
 {
-    isActive: ["auto", "desktop"],
-    trigger: ".o_field_widget[name='user_ids'] .o-autocomplete--dropdown-menu li:contains(Mitchell Admin)",
+    isActive: ["manual"],
+    trigger: ".o_field_widget[name='user_ids'] input",
```

Comment:
The initial fix was to remove that step for the onboarding ("manual") why we add it again and we also add an extra step?